### PR TITLE
Create an action to check markdown links for blog

### PR DESCRIPTION
**Pull request by Rishit-dagli**
Creates an action to check for broken links on the blogs

### CODE REVIEW

**Comment by ginomessmer** [comment]
Thanks for setting up this PR. @Rishit-dagli if you are planning to edit the readme, feel free to reuse the one I created in #2:

https://github.com/ginomessmer/mlsa-greenathon/blob/6c8ed9ed885e0f116b416ba0011ef58e96fb2f9d/README.md

**Comment by ginomessmer** [comment]
@Rishit-dagli I notice that the action doesn't run successfully, perhaps it's a formatting issue? Are you able to take a look at it?

https://github.com/ginomessmer/mlsa-greenathon/actions/runs/770727203/workflow

File: .github/workflows/markdown-links.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Check Markdown links
+
+on: push
+paths:
```
Comment (ginomessmer) [inline .github/workflows/markdown-links.yml:4]: @Rishit-dagli I notice that the action doesn't run successfully, perhaps it's a formatting issue? Are you able to take a look at it?

https://github.com/ginomessmer/mlsa-greenathon/actions/runs/770727203/workflow

**Review by ginomessmer — CHANGES_REQUESTED** [review]

File: .github/workflows/markdown-links.yml (hunk 1 above)
Comment (Rishit-dagli) [reply .github/workflows/markdown-links.yml:4]: Ah, thanks for pointing this out. I copied this from  another of my projects and seems copy-pasting messed up some formatting

**Review by ginomessmer — APPROVED** [review]

**Pull request merged**